extensions/wikia/CommunityPage/models/CommunityPageSpecialUsersModel.class.php

```diff
@@ -110,15 +111,15 @@ function () {
 				$adminIds = $this->getAdmins();
 
 				if ( !$adminIds ) {
-					return [];
+					return [ ];
 				}
 
 				$botIds = $this->getBotIds();
 				$dateTwoYearsAgo = date( 'Y-m-d', strtotime( '-2 years' ) );
 
 				$sqlData = ( new WikiaSQL() )
 					->SELECT( 'rev_user_text, rev_user, MAX(rev_timestamp) AS latest_revision' )
-					->FROM ( 'revision FORCE INDEX (user_timestamp)' )
+					->FROM( 'revision FORCE INDEX (user_timestamp)' )
 					->WHERE( 'rev_user' )->NOT_EQUAL_TO( 0 )
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "rev_user" 9 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)